### ISSUE TITLE
Improve build time analytics message shown at the start up time

### ISSUE THREAD

**Issue by sberyozkin** [quarkusio/quarkus#34284]
### Description

The following message is shown at the start up time with `3.2.0.CR1`:
```
********************************************************************************************************
********************************************************************************************************
********************************************************************************************************

* Please help improve Quarkus!
* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```

I'm not not sure having `*` to draw the user attention works well, I had it shown like this in my terminal window:
```
*********************************************************************************
***********************
*********************************************************************************
***********************
*********************************************************************************
***********************
* Please help improve Quarkus!
* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```

but also my initial reaction this was reported by some forgotten debug log message in the Quarkus code.
It looks fine after the first try, but if instead of `*` we could have a Quarkus Build Time Analytics banner it would likely look better.

The other possible problem is that if someone is steering at this message for linger than 10 secs, for example, goes to ` https://quarkus.io/usage/`, before deciding to press `n`, then the analytics will be available automatically and the question won't be asked again next time the application starts. Therefore, after the decision has been taken, either by the user selecting `y` or `n` or taken automatically, it could be useful to print a message informing the user how it can be changed/reverted if preferred later

### Implementation ideas

* Consider replacing a sequence of `*` with the banner representing the analytics feature
* Consider informing users how to change the analytics status

**Comment by geoand** [comment]
cc @brunobat

**Comment by brunobat** [comment]
@insectengine @maxandersen @paulrobinson What's your take on this?

**Comment by maxandersen** [comment]
> then the analytics decision will be done automatically and the question won't be asked again next time the application starts.

that should not be the case. if user have not pressed something we treat it as not-answered. 



> Consider replacing a sequence of `*` with the banner representing the analytics feature

I really don't like the screen wide *'s but also don't think an actual banner will be useful.

How about just:

```
***Help improve Quarkus***

* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```

> Consider informing users how to change the analytics status

quarkus.io/usage has that info - printing those instructions everytime build/devmode happens would be noisy would it not ?

**Comment by brunobat** [comment]
I would prefer to have it more visible and I'm agains displaying anything apart from the 1st dev mode execution/answer.

**Comment by maxandersen** [comment]
More visible than what ?

The current form is ultra visible and quite invasive ( as it's very wide ). 

I think we can do with less. 

Got any counter proposals to my simplified version?

**Comment by insectengine** [comment]
I agree there needs to be a visual break to indicate to the users something is happening. Instead of a long trail of asterisks that wraps oddly in smaller windows, how about 3 rows of say 20 or 30?

**Comment by brunobat** [comment]
Something like:
```
*****************************
*****************************
*** Help improve Quarkus ***

* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```
And because @sberyozkin was complaining about *, an alternative:
```
----------------------------
----------------------------
--- Help improve Quarkus ---

* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```

**Comment by insectengine** [comment]
Nicer...but how about.

----------------------------
--- Help improve Quarkus ---
----------------------------

* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
*

**Comment by insectengine** [comment]
I like where it's going... but how aobut a row of dashes, the "Help improve Quarkus", then another row?

`----------------------------
--- Help improve Quarkus ---
----------------------------

* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)`

**Comment by brunobat** [comment]
How about this?
```
----------------------------
----------------------------
--- Help improve Quarkus ---
----------------------------
* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```

**Comment by paulrobinson** [comment]
I think it looks clear, tidy and intentional.

**Comment by brunobat** [comment]
@sberyozkin @maxandersen Are you ok with this?

**Comment by maxandersen** [comment]
looks good but letse not overdo the number of lines :) 

The exact same but just 3 lines for the banner rather than 4 lines.

```
----------------------------
--- Help improve Quarkus ---
----------------------------
* Learn more: https://quarkus.io/usage/
* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n)
```
lets do it.
